src/tasks/tasks.py

```diff
@@ -707,11 +687,12 @@ def get_metrics(self, reset=False):
 
 
 @register_task('qqp', rel_path='QQP/')
+@register_task('qqp-alt', rel_path='QQP/')  # second copy for different params
```

Comment:
I think the original motivation for structuring the `-alt` tasks is that you could have attention for the `-alt`, but not the original task. Does this setup still allow us to do that?

Reply:
I think yes because modules are based on names and not the actual task class, but we should test to make sure...

Reply:
I didn't find any references for the alt class names, only for the actual task names. Can you verify this?